docs/ABOUT.md

```diff
@@ -1,41 +1,18 @@
-
-Haskell is a purely functional, lazy, statically-typed programming
-language with type inference.
+Haskell is a purely functional, lazy, statically-typed programming language with type inference. 
+It has a beautiful and minimal syntax.
 
 **Functional** means that functions are first-class data types.
 
 **Purely Functional** means (roughly) that there are no side effects.
-Every function will always return the same value for a given argument
-will do nothing else.
+Every function will always return the same value for a given argument will do nothing else.
 
-**Lazy** (a.k.a "non-strict") means that the compiler will put off
-evaluating a thing until absolutely neccessary.  This lets you safely
-do weird stuff like operating on an infinite list--the language will
-only create it up to the last value you actually use.
+**Lazy** (a.k.a "non-strict") means that the compiler will put off evaluating a thing until absolutely neccessary.
+This lets you safely do weird stuff like operating on an infinite list--the language will only create it up to the last value you actually use.
 
-**Statically-typed** means that identifiers have a type set at compile
-time--like those in Java, C++ or C#--instead of holding data of any
-type like those in Python, Ruby or JavaScript.
+**Statically-typed** means that identifiers have a type set at compile time--like those in Java, C++ or C#--instead of holding data of any type like those in Python, Ruby or JavaScript.
 
-**Type inference** means that the compiler will often figure out the
-type of an identifier by itself so you don't have to specify it.
+**Type inference** means that the compiler will often figure out the type of an identifier by itself so you don't have to specify it.
 Scala and later versions of C# both do this.
 
-Haskell syntax is beautiful and minimal.  For example, here is an
-implemention of quicksort in 6 lines:
-
-```haskell
-quicksort :: (Ord a) => [a] -> [a]
```

Comment:
Should this become the https://github.com/exercism/haskell/blob/master/docs/SNIPPET.txt ? Is the code within the size requirements?

Reply:
It's too wide for the box (42 characters max per line)